### ISSUE TITLE
Line breaks in .rst files are transferred to .pot files in a wrong way

### ISSUE THREAD

**Issue by anfelor** [sphinx-doc/sphinx#2251]
When there is a line break in the .rst files, the .pot file writes them in one line, not seperated.
When translating them, there're not split, so the same line appears twice.

See my latest [gists](https://gist.github.com/anfelor)
### How the original looks like

![http://www.pasteall.org/pic/97855](http://www.pasteall.org/pic/show.php?id=97855)
### How the translation looks like

![http://www.pasteall.org/pic/97856](http://www.pasteall.org/pic/show.php?id=97856)

**Comment by shimizukawa** [comment]
Thanks, I confirmed the issue.

```
2251
=====

Definition List
================

Term 0
   description 0

Term 1
Term 2
   description 1 and 2

Under the glossary directive
=============================

.. glossary::

   Term 0
      description 0

   Term 1
   Term 2
      description 1 and 2
```

generates html as:

![image](https://cloud.githubusercontent.com/assets/151623/12407623/2d3f8c56-be9c-11e5-989e-da1096fdd3d2.png)

I'll fix such a 'glossary' case.

**Comment by shimizukawa** [comment]
Currently(sphinx-1.3), term node possibly have multiple term text when the term is defined under glossary directive. And i18n feature requires separated term node for each term instead of single term node to create pot and to apply translation.

If we change the doctree structure to fix this issue, it become a incompatibility. So, I'll fix this issue on next major version 1.4.

**Comment by shimizukawa** [comment]
@anfelor Please try master/HEAD  or sphinx-1.4a1 (I'll release it in few hours).
